README.md

```diff
@@ -32,24 +57,37 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: stackhawk/hawkscan-action@v1.1
+    - uses: stackhawk/hawkscan-action@v1.2
       with:
         apiKey: ${{ secrets.HAWK_API_KEY }}
-        environmentVariables: APP_HOST APP_ENV
+        environmentVariables: APP_HOST, APP_ENV
       env:
         APP_HOST: http://example.com
-        APP_ENV: Preproduction
+        APP_ENV: Pre-Production
 ```
 
 ### `configurationFiles`
 
 **Optional** A list of HawkScan configuration files to use. Defaults to `stackhawk.yml`. File names can be separated with spaces, commas, or newlines.
 
+For example:
+```yaml
+jobs:
+  stackhawk-hawkscan:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: stackhawk/hawkscan-action@v1.2
+      with:
+        apiKey: ${{ secrets.HAWK_API_KEY }}
+        configurationFiles: stackhawk.yml stackhawk-extra.yml
```

Comment:
should the spaces vs. commas be consistent with the env vars above?

Reply:
I'm open to opinions! In the examples I have used all three types of listing, using spaces in one, commas in another, and newlines in the final example. All three ways work.